CODEOWNERS

```diff
@@ -1,14 +1,277 @@
 # Code owners groups and a brief description of their areas:
 # @cilium/janitors           Catch-all for code not otherwise owned
-# @cilium/api                API stability guarantees
+# @cilium/agent              Cilium Agent
+# @cilium/alibabacloud       Integration with AlibabaCloud
+# @cilium/aws                Integration with AWS
+# @cilium/azure              Integration with Azure
+# @cilium/bpf                BPF Data Path
+# @cilium/build              Building and packaging
+# @cilium/ci-structure       Continuous integration, testing
+# @cilium/cli                Commandline interfaces
+# @cilium/contributing       Developer documentation & tools
+# @cilium/docker             Docker plugin
+# @cilium/docs-structure     Documentation, examples
+# @cilium/endpoint           Endpoint package
+# @cilium/github-sec         GitHub security (handling of secrets, consequences of pull_request_target, etc.)
+# @cilium/health             Cilium cluster health tool
+# @cilium/hubble             Hubble integration
+# @cilium/ipam               IPAM
+# @cilium/kubernetes         K8s integration, K8s CNI plugin
+# @cilium/kvstore            Key/Value store: Consul, etcd
+# @cilium/loadbalancer       Load balancer
+# @cilium/loader             All related to LLVM, bpftool, Cilium loader, templating, etc.
+# @cilium/operator           Cilium operator
+# @cilium/policy             Policy behaviour
+# @cilium/proxy              L7 proxy, Envoy
+# @cilium/vendor             Vendoring, dependency management
+# @cilium/wireguard          Wireguard integration
 
 # The following filepaths should be sorted so that more specific paths occur
 # after the less specific paths, otherwise the ownership for the specific paths
 # is not properly picked up in Github.
 * @cilium/janitors
-.github/workflows/ @cilium/maintainers
-api/ @cilium/api
-pkg/apisocket/ @cilium/api
-pkg/monitor/payload @cilium/api
-pkg/policy/api/ @cilium/api
-pkg/proxy/accesslog @cilium/api
+/CODEOWNERS @cilium/contributing
```

Comment:
On release branches, we maintain a much smaller list of codeowners since for the most part it's up to the backporter, janitor and original authors to ensure that the changes make sense on the branch. Could you fix up this commit to only change the owner for `/.github/workflows/`?